src/io/export.h

```diff
@@ -35,7 +35,8 @@ enum class FileFormat {
   PNG,
   PDF,
   POV,
-  PARAM
+  PARAM,
+  Image
```

Comment:
Image doesn't really fit, but since we only support PNG at the moment, perhaps make it PNG, and either add it as a 2D file format in `initialize()` in export.cc, or add a new isImage() function.